controllers/cloud.redhat.com/clowderconfig/config.go

```diff
@@ -44,12 +44,13 @@ type ClowderConfig struct {
 		EnableAuthSidecarHook       bool `json:"enableAuthSidecarHook"`
 		KedaResources               bool `json:"enableKedaResources"`
 		PerProviderMetrics          bool `json:"perProviderMetrics"`
+		ReconciliationMetrics       bool `json:"reconciliationMetrics"`
 		DisableCloudWatchLogging    bool `json:"disableCloudWatchLogging"`
 	} `json:"features"`
 }
 
 func getConfig() ClowderConfig {
-	configPath := "/config/clowder_config.json"
+	configPath := "./config/clowder_config.json"
```

Comment:
This should not be changed - this is where clowder looks to get its config from in production

Reply:
oops